app/scenes/RootScreen/tests/saga.test.js

```diff
@@ -1,41 +1,51 @@
+/* eslint-disable sonarjs/no-duplicate-string */
 /**
  * Test sagas
  */
 
 /* eslint-disable redux-saga/yield-effects */
 
 import { takeLatest } from 'redux-saga/effects';
-import NavigationService from 'app/services/NavigationService';
+// import NavigationService from 'app/services/NavigationService';
```

Comment:
let's remove unused imports

Reply:
this has been resolved in rootscreen refactor pr